hypervisor/arch/x86/cpuid.c

```diff
@@ -158,9 +173,36 @@ static void init_vcpuid_entry(__unused struct vm *vm,
 		break;
 
 	default:
-		cpuid_subleaf(leaf, subleaf,
-				&entry->eax, &entry->ebx,
-				&entry->ecx, &entry->edx);
+		if (leaf >= 0x80000000U) {
+			/* unsupported extended cpuid level */
```

Comment:
As Anthony commented in mailinglist, do we need to change leaf > 0x8000000000 case?